tests/test_adapter.py

```diff
@@ -88,14 +114,14 @@ def test_end_to_end_file_conversion(self, _callback_post):
              ├── data
              │   ├── horizontal
              │   │   ├── east (1, 3, 3) int64
-             │   │   └── west (1, 3, 3) float64
+             │   │   └── west (1, 3, 3) uint8
              │   └── vertical
-             │       ├── north (1, 3, 3) float64
-             │       └── south (1, 3, 3) float64
+             │       ├── north (1, 3, 3) uint8
```

Comment:
This change doesn't make a ton of sense to me (but the original code doesn't either). Why is east int64 and west is uint8?

Reply:
I think this is incorrect and indicates that the scale_factor and add_offset haven't been correctly applied (whether or not they should be is up for debate, but we had a ticket to apply them, and that involves changing the data type)